Add plugin: Copy As PlainText

FinickySpider opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/FinickySpider/Obsidian-Copy-as-Plaintext

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files (as individual files, not just in the source.zip / source.tar.gz)
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the developer policies at https://docs.obsidian.md/Developer+policies, and have assessed my plugins's adherence to these policies.
- [x] I have read the tips in https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

FinickySpider commented:
Hey, just checking in. I know you guys are a small team of volunteers so this is not meant to rush or complain... I was just making sure there's not anything I'm supposed to have been doing? Or is the queue just long?🥰

joethei commented:
["fundingUrl": "https://github.com/FinickySpider",](https://github.com/FinickySpider/Obsidian-Copy-as-Plaintext/blob/49d14a0ab0850d8b7e4b0eea1b0bdfb962a05bd4/manifest.json#L9)
This is meant for links to services like _Buy me a coffee_, _GitHub sponsors_ and so on, if you don't have such a link remove this line.

[.setTitle("Copy as Plain-Text")](https://github.com/FinickySpider/Obsidian-Copy-as-Plaintext/blob/49d14a0ab0850d8b7e4b0eea1b0bdfb962a05bd4/main.ts#L65)
[Use sentence case in UI](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Use+sentence+case+in+UI), product / brand names follow the casing set by the respective owner.

[containerEl.createEl("h2", { text: "Copy As PlainText Settings" });](https://github.com/FinickySpider/Obsidian-Copy-as-Plaintext/blob/49d14a0ab0850d8b7e4b0eea1b0bdfb962a05bd4/main.ts#L231)
[Don't add a top-level heading in the settings tab, such as "General", "Settings", or the name of your plugin.](https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#Only+use+headings+under+settings+if+you+have+more+than+one+section.)

FinickySpider commented:
Hey sorry I've been super busy, thanks for the patience! I've addressed the feedback in v1.0.1 - fixed UI text casing and removed the redundant settings header. I believe everything is good now!

FinickySpider commented:
Thank you!!!
